package.json

```diff
@@ -36,6 +39,7 @@
     "aws-sdk": "2.x"
   },
   "dependencies": {
+    "http-server": "^0.11.1",
```

Comment:
What’s this now being used for?

Reply:
I don't think it's actually used for anything, the readme suggests installing it globally to run switchboard locally.

Reply:
Ok maybe it can be moved to `devDependencies` if we need to keep it - just so it doesn't get unnecessarily installed during the build.

Reply:
I think we probably need to remove as it should be installed globally because it's used from the command line, there are instructions for setting it up in the readme.

Reply:
I normally prefer to add it to `devDependencies` and add a script like `"dev-server: "http-server"` (the scripts inside a `package.json` add `{path_to_project}/node_modules/.bin` to the path before running them). That way we can have the project set it's dependency rather than hoping the global version is compatible / available.

Reply:
ok, that sounds although it's not really in the scope of this work so that's probably another pr.